app/models/user.rb

```diff
@@ -12,7 +12,7 @@ class User < ActiveRecord::Base
   has_attached_file :avatar, styles: { medium: "300x300", thumb: "100x100", landscape: "500"}, default_url: "/assets/missing.jpg"
   validates_attachment_content_type :avatar, :content_type => /\Aimage\/.*\Z/
 
-  validates_presence_of :home_city_id, :name
+  validates_presence_of :home_city_id, :nickname
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.